css/index.css

```diff
@@ -20,12 +20,17 @@ header {
   border-bottom: 1px solid #DDDDDD;
 }
 
-nav {
+header nav {
   position: absolute;
   right: 10px;
   top: 10px;
 }
 
+label[for="search"] {
+  display:block;
+  margin-bottom:.5em;
```

Comment:
Let's not mix ems and pixels. I know there's a host of reasons to prefer ems, but we've chosen to simplify and use pixels throughout fewd, so we need to be consistent.